### PR TITLE
[DNM] wip-jewel-rgw-bi-userdata

### PR DESCRIPTION
**Pull request by mattbenjamin**
that can populated/fetched via a configurable custom http header

Fixes: http://tracker.ceph.com/issues/19644
Signed-off-by: Pavan Rallabhandi <prallabhandi@walmartlabs.com>
(cherry picked from commit abca7a86c3cfbb58fafb5d057d9d6f5017a53704)
Signed-off-by: Matt Benjamin <mbenjamin@redhat.com>

### CODE REVIEW

**Comment by mattbenjamin** [comment]
http://pulpito.ceph.com/mbenjamin-2017-05-16_17:23:00-rgw-wip-jewel-udata---basic-smithi/

File: src/common/config_opts.h [hunk 1]
```diff
@@ -1453,7 +1453,10 @@ OPTION(rgw_period_push_interval_max, OPT_DOUBLE, 30) // maximum interval after e
 OPTION(rgw_swift_versioning_enabled, OPT_BOOL, false) // whether swift object versioning feature is enabled
 
 OPTION(mutex_perf_counter, OPT_BOOL, false) // enable/disable mutex perf counter
-OPTION(throttler_perf_counter, OPT_BOOL, true) // enable/disable throttler perf counter
+OPTION(throttler_perf_counter, OPT_BOOL, true) // enable/disable
+					       // throttler perf
```
Comment (yehudasa) [inline src/common/config_opts.h]: @mattbenjamin what happened to the comment here?

Comment (mattbenjamin) [inline src/common/config_opts.h]: emacs, think :(

**Comment by mattbenjamin** [comment]
@yehudasa do we want this on jewel?

**Comment by yehudasa** [comment]
@mattbenjamin not excited of getting it into jewel. That will require getting it to kraken too.

**Comment by mattbenjamin** [comment]
@yehudasa I'm fine with that

**Comment by mattbenjamin** [comment]
updated backport has been downstreamed, not accepted for Jewel, closed

**Pull request closed without merging**